File: pyxlib/test/csharp/unit_test/unit_test/unit_test.cs

public class unit_test
{
	static int Main()
	{
        PYXLibInstance.initialize("unit_test (C#)");

        // Run unit tests
        bool bSuccess = TestFrame.getInstance().testUnit();

        PYXLibInstance.destroy();

        return bSuccess ? 0 : 1;
    }
}

File: pyxlib/test/cpp/unit_test/unit_test/unit_test.cpp
// unit_test.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include "pyxlib_instance.h"
#include "pyxis/utility/tester.h"

int _tmain(int argc, _TCHAR* argv[])
{
	PYXLibInstance pyxLib("unit_test (C++)");

	// Run unit tests
	bool bSuccess = TestFrame::getInstance().testUnit();

	return bSuccess ? 0 : 1;
}
